lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -129,6 +130,14 @@ def add_customer_data(post, payment, options)
         post[:payment][:name] = payment.name
         post[:payment][:email] = options[:email] || "unspecified@example.com"
         post[:payment][:document] = options[:document]
+        post[:payment][:birth_date] = options[:birth_date] if options[:birth_date]
+      end
+
+      def add_customer_responsible_person(post, payment,  options)
+        post[:payment][:responsible] = {}
```

Comment:
Hah ...or I could just be completely wrong! I guess it doesn't like adding new elements two-deep. Merging!